### DISCUSSION TITLE
Conversion optimisation on the checkout page

### DISCUSSION THREAD

**Discussion by janbaykara** [commonknowledge/leftbookclub#232] [General]
After viewing a plan, the [checkout / shipping page](https://leftbookclub.com/confirm-shipping/2/prod_LcPK6pYSDxDh9k/) is where the most people drop off the checkout flow (the "logged in" drop off of 47.5%). This isn't that unusual, to be fair: [ecommerce websites report a 2/3 drop off on their checkout page](https://zoolatech.com/blog/best-approaches-for-shopping-cart-page-optimization/) frequently.

<img width="573" alt="Screenshot 2022-05-20 at 10 19 53" src="https://user-images.githubusercontent.com/237556/169497292-3acef5ed-a5ea-4963-8073-94a7bbf1d91a.png">

I've done a bit of light research on how we could improve the conversion rate from this point on, will note some ideas here:

### Trust badges: "You can cancel at any time"

Being upfront about aspects of the membership 'contract' might ease concerns about what is otherwise a bit black and white decision.

- You can cancel at any time
- Secure payments
- Contact us any time
- ??? anything else we can say

<img width="1028" alt="Screenshot 2022-05-20 at 10 40 55" src="https://user-images.githubusercontent.com/237556/169501156-cfbf0324-243c-46e5-b09d-6df90f07cc0b.png">

<img width="294" alt="Screenshot 2022-05-20 at 10 51 13" src="https://user-images.githubusercontent.com/237556/169503012-a7640b29-b44a-4795-8ca5-66b572366600.png">

### Customer testimonials / reviews

It could help build some trust if we throw up a few faces and quotes from existing members, saying how it's going for them.

![](https://www.chargebee.com/blog/wp-content/uploads/2021/06/customer-testimonials-at-checkout.png)

### Book photos and better language

- *Add a picture of the next book for this plan* saying "your next book"
- Better checkout copy. At the moment the copy asks you to pick it, rather than telling you what comes next. We could say:
  - Customer testimonials (above) 
  - The next book that's coming, when it's coming, how it will get to you
  - What else we'll send (promo code email?)
  - Replace "confirm details" header -> "You're Joining the Left Book Club", and maybe add a photo and some motivational copy about what historically / recently it means to join the left book club?
  - Could we guess at an estimated delivery date/range?
 
<img width="443" alt="Screenshot 2022-05-20 at 10 54 19" src="https://user-images.githubusercontent.com/237556/169503560-27a365a2-f247-42ff-89e3-6dbb9d43ae8a.png">

### Add breadcrumbs

It might help to know what step the customer is on so they know if there's much faff left (spoiler alert: there isn't!)

<img width="281" alt="Screenshot 2022-05-20 at 10 50 39" src="https://user-images.githubusercontent.com/237556/169502898-c8bb7500-7162-4e31-9f30-d41c6b754d6a.png">

<img width="648" alt="Screenshot 2022-05-20 at 10 34 41" src="https://user-images.githubusercontent.com/237556/169500083-9c99215c-a8b2-40a0-bd72-f6d5099c8d48.png">

### ***Simplifying*** checkout: removing the header and footer

Amazon and a few other ecommerce websites do this. By removing distractions (like header links, e.g. to the books page) the logic is that you canalise people's attention into the current activity: pick your shipping, continue.

We could try this as an experiment, using A/B split testing, to see if one yields higher conversions than the other. I have no idea if this will be effective or not, so it could be worth a try.

We could even take this further and actually slightly de-emphasise the product information so that the primary user action is made clear: pick your shipping country and click continue. Perhaps as part of this, we get rid of the white background for the product info and move the price calculation to the right hand column.

### Reassurance about promo codes and payment

Some people are showing up hoping to enter their promo code. We could add a message on this page saying "enter it on the next page" or something, to reassure them.

The stretch on this is that we could actually let people put their promo code in on the checkout page, to "lock them in", but I think we could start by just mentioning.

<img width="213" alt="Screenshot 2022-05-20 at 11 06 14" src="https://user-images.githubusercontent.com/237556/169505765-f32a9407-1cd7-4b6a-99a4-22ba5e7f6104.png">

### On-exit popup with a discount

Could capture a few folk who give up, if we offered a first month free discount or something.

![](https://www.chargebee.com/blog/wp-content/uploads/2021/06/exit-intent-popups.png)

**Comment by janbaykara** [comment]
For example, @melaniepatrick what do you think about us showing the next delivered book for a plan on the checkout?

<img width="780" alt="Screenshot 2022-05-20 at 14 19 43" src="https://user-images.githubusercontent.com/237556/169539607-fd9ab5ea-2b4a-4072-b845-ed63c972ec8e.png">

Compared to the current page:

<img width="889" alt="Screenshot 2022-05-20 at 14 36 36" src="https://user-images.githubusercontent.com/237556/169539708-8f9ac1f7-c393-4770-9d68-cc9e2d767a42.png">

In the example above, we don't display the product text. We could continue to include that (see below) although the page gets a bit long:

<img width="909" alt="Screenshot 2022-05-20 at 14 35 00" src="https://user-images.githubusercontent.com/237556/169539549-f8a9d6e4-d904-461c-9575-3f51ed2924c9.png">

**Comment by janbaykara** [comment]
@melaniepatrick here is how breadcrumbs might look, as an "orientation" / "what next?" device:

<img width="914" alt="Screenshot 2022-05-20 at 14 58 53" src="https://user-images.githubusercontent.com/237556/169544166-08553add-98a5-4e70-9323-950029b33732.png">

**Comment by melaniepatrick** [comment]
I think this is great if we are able to acccurately specify the book for
each sub. thanks!

**Reply by janbaykara** [reply]
@melaniepatrick The way this works is that at the [Stripe product level](https://dashboard.stripe.com/products/prod_Lcj16IInOc32pb), I've added a `book_types` metadata item with the relevant `type` option that corresponds to the Shopify book `type`. So if the user has selected a stipe product with `book_types` set (i.e. the Classics or Contemporary products), it will look for the most recent Shopify book for that type. If the stripe product doesn't have a `book_types` set (i.e. Solidarity, All Books), then it will just pull in the most recent Shopify book without filtering by type.

I'll push this to https://lbc-staging.com for you to have a look!

**Comment by melaniepatrick** [comment]
So just read all of this, thanks for many excellent suggestions:
1) I really like replacing the current product text with the photo of the book they will get
2) I agree that's it's essential to remind them that they can cancel at any time. So that last sentence of the product description is very important to retain. I think it should be positioned very clearly. Perhaps it needs to go in the data in the left hand column, rather than below the photo? Not sure.
3) I don't think we need the text below the photo, I agree it becomes too long and distracting

**Comment by melaniepatrick** [comment]
So just to clarify, 2. should use the wording, either monthly or annual, from the existing subscription descriptions (ie Monthly subscriptions automatically renew. You can cancel at any time.)

**Comment by melaniepatrick** [comment]
Definitely like the text 'You're joining the Left Book Club', much better.

**Comment by melaniepatrick** [comment]
Breadcrumbs looks good also. However what happens to this process if people press 'login' rather than 'join'? I think that is what has happened in quite a few of the cases I've been watching on POsthog. So people set up accounts and then seem to pick their sub afterwards, or try to? Could breadcrumbs potentially confuse them?

**Reply by janbaykara** [reply]
Good spot! We actually won't show them the "Create account" breadcrumb if they are already logged in.

Interesting that you notice people sneakily creating an account before registering.

**Reply by melaniepatrick** [reply]
Yes. I actually suspect they do it because 'Join' is quite small and not highlighted. So they go to where login is, thinking that is how they sign up.

**Reply by melaniepatrick** [reply]
I don't really know how to fix that. As I can understand why the menu is the way it is. We need to make it obvious where people redeem vouchers. And we can't really highlight more than one thing in yellow. But on Gemma's original layout she put 'JOIN' at the end in yellow

**Reply by janbaykara** [reply]
Oh, I see. Yes, that makes sense. Well, I have an idea: why don't we actually just replace Redeem with Join?

People who have a gift card are already having their membership paid, and their gift giver has the instructions and even a special URL for them to visit. There are far fewer gift cards bought than regular memberships, so it is taking up valuable real estate that could be put to better use, and "join" is probably the primary ask we have of visitors to the website. We could put "redeem" in the left hand nav menu instead!

**Reply by melaniepatrick** [reply]
ok this sounds sensible.

**Comment by melaniepatrick** [comment]
Great idea to add promo code text. 
I'm happy with:
'Got a promo code? Add it at the next step'

I'm interested in the idea of offering them a discount too. However I think going straight to giving them the first month free might be a bit much. Let's see how we go with the current changes and keep this to try later.

**Comment by janbaykara** [comment]
So here's where we're at with this now:

<img width="927" alt="Screenshot 2022-05-20 at 17 10 46" src="https://user-images.githubusercontent.com/237556/169568942-a3dcc01b-4224-42c6-9f67-581002d69ec4.png">

compared to 

<img width="961" alt="Screenshot 2022-05-20 at 17 10 37" src="https://user-images.githubusercontent.com/237556/169569104-e5577d18-ee40-49a6-8848-362b4d80c4d2.png">

I'm not sure if the promo code and cancellation text positioning is effective, or if they'd be better of as bullet points or simple paragraph text just above the "Continue" button, or something else?

**Reply by janbaykara** [reply]
Here is a live link to the staging site showing how this works: https://lbc-staging.fly.dev/confirm-shipping/6/prod_LPl1Or1YFIGUDl/?

**Reply by janbaykara** [reply]
CC @melaniepatrick ^

**Reply by janbaykara** [reply]
OK, I'm checking out for the week but can deploy these changes as they are if you give the OK @melaniepatrick.

**Reply by melaniepatrick** [reply]
Thanks I am generally happy with this. It is much better than before.
1) is the breadcrumb trail accurate? At the moment although we are viewing shipping, 'Select Plan' is still underlined and shipping is grey. I think underlining is clearer than greying out, if I had to choose between the 2 options. Then when we go to the next page (the account set up page) then we lose the breadcrumb trail? Should we keep it? 
2) I'm happy with positioning of cancel text and code info 
3) Screenshot of current live layout shows that if you are logged in you get a tick box warning you that it is updating your sub. I think this is very good and clear and we should retain it. Just wanted to check that we are not losing that feature with these changes you are making.? 
4) general question: when someone renews or changes their subscription, is their order sent through to shopify? Is the trigger for an order to be sent to shopify the fact that someone is using the checkout? Basically, if we are telling people what book they are going to get, I will need to be very careful to make sure that we mail renewed people as well as new members

**Reply by janbaykara** [reply]
1. Breadcrumbs:
  - [x] Let's add breadcrumbs to the signup page
  - [x] So we'll make select plan grey, shipping underlined
2. Cool
3. It's kept :)
4. Yes, shopify orders are created whenever someone goes through checkout (or redeems a gift code), so renewals will also trigger a shopify order

I'm going to make the breadcrumb changes and then deploy this, and then we can continue to iterate.

**Reply by melaniepatrick** [reply]
Fab thank you! Have a good weekend.

**Comment by melaniepatrick** [comment]
sorry Jan had to leave work early today just going to read through this now
